Remotion/Web/ClientScript/Scripts/TreeView.ts

```diff
@@ -103,14 +103,15 @@ class WebTreeView
 
   private static OnKeyDown (event: KeyboardEvent, treeView: HTMLElement): void
   {
-    const treeNodes = Array.from (treeView.querySelectorAll<HTMLElement> ('li[role=treeitem]'));
+    const treeNodes = Array.from (treeView.querySelectorAll<HTMLElement> ('a[role=treeitem]'));
+    const parentTreeNodeListItems = Array.from(treeNodes.map<HTMLElement>(a => a.closest('li')!));
```

Comment:
I'm not sure we need the LI-mappings this way instead of just selecting them as we need them via parent and child navigation. Right now, it feels overly complex and might not give us a true performance benefit that offsets the complexity.

Reply:
Yeah, I agree.